pulp_2to3_migration/app/models/base.py

```diff
@@ -68,93 +70,143 @@ def get_missing_resources(self):
         """
         ret = {}
         if self.plan_view.missing_repositories:
-            ret['repositories'] = self.plan_view.missing_repositories
-        if self.plan_view.missing_importers:
-            ret['importers'] = self.plan_view.missing_importers
-        if self.plan_view.missing_distributors:
-            ret['distributors'] = self.plan_view.missing_distributors
+            ret['repositories'] = self.plan_view.repositories_missing_repositories
+        if self.plan_view.repositories_missing_importers:
+            ret['repositories_missing_importers'] = self.plan_view.missing_importers
+        if self.plan_view.repositories_missing_distributors:
+            ret['repositories_missing_distributors'] = \
+                self.plan_view.repositories_missing_distributors
```

Comment:
Does it make sense to make the ```missing_resources``` plugin specific as well?

Reply:
If it's more convenient to do so because you refactor to have plans by plugin, sure, go for it.
Otherwise, as I usually do, I'd postpone until it's needed.

Reply:
I don't see an urgent need so we can just postpone, but I didn't know if it might be useful.